script/team_pages.py

```diff
@@ -11,10 +11,21 @@
     'sppl',
 ]
 
+theme_colors = [
+    'green',
+    'yellow',
+    'orange',
+    'blue'
+]
+
 teams = filter(lambda t: t['page_slug'] not in dedicated_team_page_slugs, teams)
 
-for id, name, slug in [ (d['id'], d['name'], d['page_slug']) for d in teams if d['id'] != 3]:
-    filename = '_teams/{}.md'.format(slug)
-    print(filename)
+for i,d in enumerate(teams):
+    if d['id'] == 3:
+        break
```

Comment:
@s-kennedy `break` will terminate the for loop, to just skip id==3 you'd want to say `continue`